muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/guider/dialogs/student/tabs/state-of-studies/ceepos.tsx

```diff
@@ -344,7 +344,13 @@ class Ceepos extends React.Component<CeeposProps, CeeposState> {
                         ) : null}
 
                         {/* We show "Complete order" button only if logged in user has COMPLETE_ORDER permission */}
-                        {this.props.status.role === "ADMINISTRATOR" ? (
+                        {(
+                          this.props.status.roles
+                            ? this.props.status.roles.includes(
+                                Role.Administrator
+                              )
+                            : false
```

Comment:
Unnecessary null/undefined check as roles is always empty list by default. Can use includes checking straight